scout/server/blueprints/genes/views.py

```diff
@@ -50,11 +50,9 @@ def gene(hgnc_id=None, hgnc_symbol=None):
 def api_genes():
     """Return JSON data about genes."""
     query = request.args.get("query")
-    if query is None:
-        return jsonify({"code": 400, "message": "missing 'query' param in request"})
+    if query is None or query.replace("-", "").isalnum() is False:
```

Comment:
Ok, yes, that should work. I can't think of any gene with other chars in the symbol, and the docs seem good.